block/manager.go

```diff
@@ -563,6 +557,34 @@ func (m *Manager) HeaderSubmissionLoop(ctx context.Context) {
 	}
 }
 
+func (m *Manager) handleEmptyDataHash(ctx context.Context, header *types.Header) {
+	headerHeight := header.Height()
+	if bytes.Equal(header.DataHash, dataHashForEmptyTxs) {
+		var lastDataHash types.Hash
+		var err error
+		var lastData *types.Data
+		if headerHeight > 1 {
+			_, lastData, err = m.store.GetBlockData(ctx, headerHeight-1)
```

Comment:
[non blocking] Is err here due to out of order sync? Should we log it at debug level to help trace the execution path in future?